submissions/BukkyOyetimehin/HTML/product.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
```

Comment:
Kindly do this work on your existing product page that you created before during this HTML task. You don't have to create a new product page file

Reply:
@adaugochi The present product page is actually an update of the former product page which I did during the HTML task.  That was what I meant.to write.

Reply:
ok